.env.example

```
@@ -0,0 +1,15 @@
+# RUBIN node — dev environment template
+# Copy to .env and fill in values before running locally.
+# For production/systemd see: operational/systemd/rubin-node.env.example
+
+# wolfCrypt shim (required for PQ signatures in non-dev builds)
+RUBIN_WOLFCRYPT_SHIM_PATH=
+RUBIN_WOLFCRYPT_SHIM_SHA3_256=
+RUBIN_WOLFCRYPT_STRICT=1
+
+# Node config
+RUBIN_DATADIR=~/.rubin/data
+RUBIN_CHAIN=devnet
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RUBIN_CHAIN key should go before the RUBIN_DATADIR key

(UnorderedKey)